CMakeLists.txt

```diff
@@ -1600,7 +1750,7 @@ elseif(MSVC)
   # may include cmath first.
   target_compile_definitions(QueenMaryDsp PUBLIC _USE_MATH_DEFINES)
 endif()
-target_include_directories(QueenMaryDsp SYSTEM PUBLIC lib/qm-dsp lib/qm-dsp/include)
+target_include_directories(QueenMaryDsp SYSTEM PUBLIC lib/qm-dsp lib/qm-dsp/include /usr/include/x86_64-linux-gnu/qt5/QtCore /usr/include/x86_64-linux-gnu/qt5 src )
```

Comment:
This hard-coded platform dependency looks suspicious!?

Reply:
Ups, it is. Thank you for noticing.
This is a debug leftover.